plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -1174,66 +1169,41 @@ func isPodBestEffort(pod *api.Pod) bool {
 // CheckNodeMemoryPressurePredicate checks if a pod can be scheduled on a node
 // reporting memory pressure condition.
 func CheckNodeMemoryPressurePredicate(pod *api.Pod, meta interface{}, nodeInfo *schedulercache.NodeInfo) (bool, []algorithm.PredicateFailureReason, error) {
-	node := nodeInfo.Node()
```

Comment:
Not returning an error is even better. Basically when this can happen is data race between two different caches when in one node was already removed and from this one not yet. Previously, the whole scheduling would end with error in this case (which basically shouldn't, but it's not a disaster as it will be retried).
In this case, this basically will result in saying that the node doesn't satisfy constraints which is exactly what we need.